_posts/2016-08-18-Rust-1.11.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title: "Announcing Rust 1.11"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.11. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+As always, you can [install Rust 1.11][install] from the appropriate page on our
+website, and check out the [detailed release notes for 1.11][notes] on GitHub.
+1109 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/nightly/RELEASES.md#version-1110-2016-08-18
```

Comment:
Gah, I am too used to doc.rust-lang.org :)

These _usually_ link to stable, but the release notes didn't make it in this time, so I typed in nightly manually. I'll fix this to make sure it's a real link.

On Aug 17, 2016, 22:12 -0400, Alex Crichton <notifications@github.com>, wrote:
>
> In _posts/2016-08-18-Rust-1.11.md (https://github.com/rust-lang/blog.rust-lang.org/pull/119#discussion_r75239240):
>
> > @@ -0,0 +1,224 @@ > +--- > +layout: post > +title: "Announcing Rust 1.11" > +author: The Rust Core Team > +--- > + > +The Rust team is happy to announce the latest version of Rust, 1.11. Rust is a > +systems programming language focused on safety, speed, and concurrency. > + > +As always, you can [install Rust 1.11][install] from the appropriate page on our > +website, and check out the [detailed release notes for 1.11][notes] on GitHub. > +1109 patches were landed in this release. > + > +[install]: https://www.rust-lang.org/install.html > +[notes]: https://github.com/rust-lang/rust/blob/nightly/RELEASES.md#version-1110-2016-08-18
>
> We have a "nightly" branch?
>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub (https://github.com/rust-lang/blog.rust-lang.org/pull/119/files/6c8c96acf537ec5c65ddcd3ce62bded37a16ca55#r75239240), or mute the thread (https://github.com/notifications/unsubscribe-auth/AABsihOD1K_UR5GduORcnSLJHWSPLDd5ks5qg7-ggaJpZM4JnCRM).
>
>
>